src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -665,14 +665,14 @@ App::DocumentObjectExecReturn *Cone::execute()
             //Build a cylinder
             BRepPrimAPI_MakeCylinder mkCylr(Radius1.getValue(),
                                             Height.getValue(),
-                                            2.0 * M_PI);
+                                            Base::toRadians<double>(Angle.getValue()));

```

Comment:
Why don't you use `angle` here?

Reply:
Because I don't want to apply the modified angle.

Reply:
But it will break the OCC 7.7.2 compatibility until another fix was pushed. Shouldn't the `angle` removing done in a subsequent PR?

Reply:
Ok, great. So the lines 656-661 should be removed.

Reply:
That removal is part of PR #17306